gp-templates/header.php

```diff
@@ -4,7 +4,11 @@
 	<meta http-equiv="Content-type" content="text/html; charset=utf-8" />
 	<title><?php echo gp_title(); ?></title>
 
-	<?php gp_head(); ?>
+	<?php
+	// Enqueue the base style so we don't have to load it manually on each page.
+	gp_enqueue_style( 'gp-base' );
```

Comment:
Could, but leaving it in the header template makes it easier for sites to modify if required for their own custom templates.